### PR TITLE
USDCUSD Oracle Config

### PR DESCRIPTION
**Pull request by sissnad**
### Description

This PR adds 4 price sources for the USDCUSD oracle. Oracle parameters have been calibrated:

```
aggregation:
    mid: 
      maxPercentageDeviation: 0.01
      maxPercentageBidAskSpread: 0.005

reporter:
    blockBased:
        minReportPriceChangeThreshold: 0.0005
```

### Related Issues

[Update USDC/USD config parameters](https://github.com/mento-protocol/mento-general/issues/159)
[Add WhiteBIT exchange adapter for USDC/USD](https://github.com/mento-protocol/mento-general/issues/158)

### CODE REVIEW

File: packages/helm-charts/oracle/USDCUSD.yaml [hunk 1]
```diff
@@ -16,14 +16,19 @@ oracle:
...
+        {exchange: 'BINANCEUS', symbol: 'USDCUSD', toInvert: false}
       ],
       [
-        { exchange: 'GEMINI', symbol: 'USDCUSD', toInvert: false }
+        {exchange: 'COINBASE', symbol: 'USDTUSDC', toInvert: true},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
+      ],
+      [
+        {exchange: 'WhiteBIT', symbol: 'USDCUSDT', toInvert: false},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
```
Comment (ghost) [inline packages/helm-charts/oracle/USDCUSD.yaml:27]: was the WhiteBIT:USDTUSD rate bad?

File: packages/helm-charts/oracle/USDCUSD.yaml [hunk 2]
```diff
@@ -16,14 +16,19 @@ oracle:
...
       ]
     ]"
```
Comment (ghost) [inline packages/helm-charts/oracle/USDCUSD.yaml:29]: Interesting that OKX and KuCoin rates are more unstable although a higher volume seems to be traded there

File: packages/helm-charts/oracle/USDCUSD.yaml [hunk 3]
```diff
@@ -3,8 +3,8 @@ oracle:
   aggregation:
     mid:
       maxExchangeVolumeShare: 1
-      maxPercentageDeviation: 0.05
-      maxPercentageBidAskSpread: 0.025
+      maxPercentageDeviation: 0.01
+      maxPercentageBidAskSpread: 0.005
```
Comment (ghost) [inline packages/helm-charts/oracle/USDCUSD.yaml:7]: how did you arrive at 0.5% here?

File: packages/helm-charts/oracle/USDCUSD.yaml [hunk 4]
```diff
@@ -16,14 +16,19 @@ oracle:
...
+        {exchange: 'WhiteBIT', symbol: 'USDCUSDT', toInvert: false},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
       ]
     ]"
   minPriceSourceCount: 2
   reportStrategy: BLOCK_BASED
   reporter:
     blockBased:
-      minReportPriceChangeThreshold: 0.005 # 0.5%
+      minReportPriceChangeThreshold: 0.0005 # 0.5%
```
Comment (ghost) [inline packages/helm-charts/oracle/USDCUSD.yaml]: Can you update or remove the comment: it's 0.05% instead of 0.5% now I guess

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/helm-charts/oracle/USDCUSD.yaml (hunk 2 above)
Comment (tobikuhlmann) [reply packages/helm-charts/oracle/USDCUSD.yaml:29]: Rates and config look good to me :)

**Comment by ghost** [comment]
Rates and config parameters look good to me :)

File: packages/helm-charts/oracle/USDCUSD.yaml (hunk 1 above)
Comment (sissnad) [reply packages/helm-charts/oracle/USDCUSD.yaml:27]: Here is the distribution of the deviation from average using WhiteBIT USDTUSD:

image


And here using Coinbase USDTUSD (sorry, for being too lazy to use same scale and colours...)

!image

I believe the differences come from trading volumes at WhiteBIT USDTUSD has 2m daily at Coinbase it is 80m

Comment (ghost) [reply packages/helm-charts/oracle/USDCUSD.yaml:27]: ok wow 😅

File: packages/helm-charts/oracle/USDCUSD.yaml (hunk 2 above)
Comment (sissnad) [reply packages/helm-charts/oracle/USDCUSD.yaml:29]: Yes, they were my initial candidates. But maybe WhiteBIT has better merket makers...

File: packages/helm-charts/oracle/USDCUSD.yaml (hunk 3 above)
Comment (sissnad) [reply packages/helm-charts/oracle/USDCUSD.yaml:7]: I think for most stables market makers will try to keep a b/a spread sub 10bps. In that sense if the b/a spread is at 50bps I guess either the market maker is offline or there is a lot of uncertainty in the market

Comment (ghost) [reply packages/helm-charts/oracle/USDCUSD.yaml:7]: ok thanks

**Review by ghost — APPROVED** [review]

**Pull request merged**